packages/lib/src/utils/base64.ts

```diff
@@ -35,7 +50,11 @@ const base64 = {
             return false;
         }
 
-        return window.btoa(window.atob(pDataStr)) === pDataStr;
+        try {
+            return window.btoa(window.atob(pDataStr)) === pDataStr;
+        } catch (e) {
+            return false;
+        }
```

Comment:
I believe this check will fail 16 bit UTF char. This makes that we will never reach `b64DecodeUnicode` when calling in `decode`.